sdk/core/logger/src/logger.ts

```diff
@@ -30,12 +32,20 @@ AzureLogger.log = (...args) => {
  * - warning
  * - error
  */
-export type AzureLogLevel = "verbose" | "info" | "warning" | "error";
+const AZURE_LOG_LEVELS = ['verbose', 'info', 'warning', 'error'] as const;
```

Comment:
`as const` is unnecessary now since I removed the fancy type arithmetic to compute the AzureLogLevel type.